src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -55,13 +55,16 @@
 import org.gradle.testing.jacoco.plugins.JacocoTaskExtension;
 import org.sonarsource.scanner.api.Utils;
 
+import static java.util.Arrays.asList;
+
 /**
  * A plugin for analyzing projects with the <a href="http://redirect.sonarsource.com/doc/analyzing-with-sq-gradle.html">SonarQube Runner</a>.
  * When applied to a project, both the project itself and its subprojects will be analyzed (in a single run).
  * Please see the “SonarQube Runner Plugin” chapter of the Gradle User Guide for more information.
  */
 public class SonarQubePlugin implements Plugin<Project> {
 
+  private static final Pattern TEST_RESULT_FILE_PATTERN = Pattern.compile("TESTS?-.*\\.xml");
```

Comment:
But if Gradle would produce a different pattern, the SurefireJavaParser wouldn't catch them anyway currently as only the pattern here is considered.